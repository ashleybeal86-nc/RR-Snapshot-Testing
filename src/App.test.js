import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders text AshleyBeal", () => {
  render(<App />);
  const nameElement = screen.getByText(/Ashley Beal/i);
  expect(nameElement).toBeInTheDocument();
});
