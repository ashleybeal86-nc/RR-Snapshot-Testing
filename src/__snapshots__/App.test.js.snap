// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<h1>
  Snapshot Testing
</h1>
`;
